### PR TITLE
DFD-133 v1 of season score

### PR DESCRIPTION
**Pull request by cha0sg0d**
Creates new Graph queries for the EGP. Fetch Season Score, Badges, Wallbreakers.

Also includes a small refactor of files in the `client/src/Backend/Network/GraphApi` folder. Relies on https://github.com/dfdao/packages/pull/35/files as well

Screen Shot 2022-08-19 at 4 00 39 PM

Season Score (in screenshot) is the players best time on each grand prix (inverted by 24 hours in seconds) plus any score they received from badges

### CODE REVIEW

File: src/Backend/Network/GraphApi/GrandPrixLeaderboard.ts [hunk 1]
```diff
@@ -14,43 +14,35 @@ export async function loadArenaLeaderboard(
...
       moves
-   }
+   }    
     gameOver
     endTime
     duration
   }
 }
 `;
+console.log(`query`, QUERY);
```
Comment (cha0sg0d) [inline src/Backend/Network/GraphApi/GrandPrixLeaderboard.ts]:
```suggestion
```

**Review by cha0sg0d — COMMENTED** [review]
remove console log

File: src/Frontend/Views/Portal/PortalMainView.tsx [hunk 2]
```diff
@@ -101,6 +102,13 @@ export function PortalMainView() {
         <TopBar>
           <TitleContainer>
             <Title onClick={() => history.push('/portal/home')}>Home</Title>
+            <Button onClick={async () => {const a = await loadSeasonLeaderboard()}}>
```
Comment (0xhank) [inline src/Frontend/Views/Portal/PortalMainView.tsx]:
```suggestion
```

File: src/Backend/Network/GraphApi/SeasonLeaderboardApi.ts [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  for (const [player, grandPrixResults] of Object.entries(seasonPlayers)) {
+    const seasonScore: SeasonScore = {
+      player,
+      score: grandPrixResults
+        .map((result) => calcSeasonScore(result))
+        .reduce((prev, curr) => prev + curr),
+    };
+    seasonScores.push(seasonScore);
+  }
+  console.log(`season Scores`, seasonScores);
```
Comment (0xhank) [inline src/Backend/Network/GraphApi/SeasonLeaderboardApi.ts:148]:
```suggestion
```

File: src/Backend/Network/GraphApi/GrandPrixApi.ts [hunk 4]
```diff
@@ -8,49 +8,48 @@ import {
...
     gameOver
     endTime
     duration
   }
 }
 `;
+console.log(`query`, QUERY);
   const rawData = await getGraphQLData(QUERY, process.env.GRAPH_URL || 'localhost:8000');
-
+  console.log('arenaData', rawData);
```
Comment (0xhank) [inline src/Backend/Network/GraphApi/GrandPrixApi.ts:45]:
```suggestion
```

File: src/Backend/Network/GraphApi/GrandPrixApi.ts [hunk 5]
```diff
@@ -8,49 +8,48 @@ import {
...
       moves
-   }
+   }    
     gameOver
     endTime
     duration
   }
 }
 `;
+console.log(`query`, QUERY);
```
Comment (0xhank) [inline src/Backend/Network/GraphApi/GrandPrixApi.ts:43]:
```suggestion
```

File: src/Backend/Network/GraphApi/SeasonLeaderboardApi.ts [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+        configHash
+        lobbyAddress
+        winners {
+          address
+        }
+        duration
+      }
+    }
+    `;
+    console.log(QUERY);
```
Comment (0xhank) [inline src/Backend/Network/GraphApi/SeasonLeaderboardApi.ts:44]:
```suggestion
```

**Review by 0xhank — CHANGES_REQUESTED** [review]
remove these fake buttons from the top bar

File: src/Frontend/Views/Portal/PortalMainView.tsx (hunk 2 above)
Comment (0xhank) [reply src/Frontend/Views/Portal/PortalMainView.tsx]: delete all these buttons

**Pull request merged**